### PR TITLE
Command tweaks

### PR DESCRIPTION
**Pull request by DigiTechs**
Most of these tweaks make documentation of commands easier for bot developers who want to do things their own way.

### CODE REVIEW

File: src/Discord.Net.Commands/Attributes/DescriptionAttribute.cs [hunk 1]
```diff
@@ -12,7 +13,8 @@ public DescriptionAttribute(string text)
         }
     }
 
-    [AttributeUsage(AttributeTargets.Method)]
+    // Brief summary of method/module
```
Comment (khionu) [inline src/Discord.Net.Commands/Attributes/DescriptionAttribute.cs]: Uhhhh.... Synopsis was intended to be the Longer summary, as Description is for Parameters, too.... A Synopsis is something intended to give an overview of something, beginning to end.

Comment (DigiTechs) [inline src/Discord.Net.Commands/Attributes/DescriptionAttribute.cs]: Synopsis is usually defined as 'a brief summary' (in contrast to description, which is defined as 'a spoken or written account of a person, object, or event' - basically a 'full summary') which is why I changed the meanings about.

**Comment by khionu** [comment]
LGTM

**Comment by RogueException** [comment]
Modules now being accessible via `Commands` now makes it less clear imo
Also why are we using `Synopsis`? Was this a discussion I missed? Feels less clear than something like `Summary`, especially when you guys already arent able to decipher its meaning at first glance.

**Comment by khionu** [comment]
I suggested it, asked twice if anyone had any issues with the phrasing :/ I thought I was clear at intended usage....

**Comment by DigiTechs** [comment]
I can add the Modules enumerable back if that would be preferred. I also renamed Synopsis to Summary to make the differences between it and Description clear.

**Comment by RogueException** [comment]
I'd say revert https://github.com/RogueException/Discord.Net/pull/178/commits/3856d435876b4aab3833ba09fe2c878071be3a0a and it should be good.

We dont use ILookup anywhere in the lib, having it here isn't consistent (and I dont like it much anyway :P).

After that's fixed, LGTM.

**Pull request merged**